crates/compiler/builtins/roc/Dict.roc

```diff
@@ -27,15 +27,15 @@ interface Dict
     ]
 
 ## A [dictionary](https://en.wikipedia.org/wiki/Associative_array) that lets you
-## can associate keys with values.
+## associate keys with values.
 ##
 ## ### Inserting
 ##
 ## The most basic way to use a dictionary is to start with an empty one and
 ## then:
-## 1. Call `Dict.insert` passing a key and a value, to associate that key with
+## 1. Call [Dict.insert] passing a key and a value, to associate that key with
```

Comment:
I'm pretty sure the brackets are for types. I've played with this a bit and they link to the type definition. So all the functions I have been using ``. I noticed both were being used in the docs so I wasn't sure.

Reply:
`[List.first]` does work as expected:
- [line in code](https://github.com/roc-lang/roc/blob/f0b65048cd83868fcc81aed55eb804ae96c95f5a/crates/compiler/builtins/roc/List.roc#L203)
- [list docs](https://www.roc-lang.org/builtins/List)

Does `` create a link?
If `[Dict.insert]` links to `Dict` that sounds like a bug.